src/App.js

```diff
@@ -0,0 +1,33 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
- [ ] Please avoid disabling linter rules, we use them to ensure a consistent code format and it is useful to help you adapt some best practices and conventions! 🤓 🔧